index.md

```diff
@@ -68,7 +68,7 @@ eventbrite:           # optional: clave alfanumérica de registro en Eventbrite,
 {% comment %}
   PÚBLICO
 
-  Explica quién es tu público. (En particular, cuenta a los lectores si el taller esta abierto sólo a personas de una institución o grupo en particular).
+  Explica quién es tu público. (En particular, cuenta a los lectores si el taller esta abierto sólo a personas de una institución o grupo en particular). Asegúrate de modificar el texto debajo del tipo de taller que estás organizando (swc, dc o lc).
```

Comment:
This was the same as the above comment, I did not initially noticed that this was linked to another file. How about including a paragraph like the one below:
_Para editar el párrafo introductorio general deberás modficar el archivo_ `who.html`. _Este archivo lo puedes encontrar navegando a la carpeta includes y luego selecciona la carpeta para el tipo de taller que estás organizando (sw, dc, o lc)._
This is probably not necessary for people who are familiar with this sort of website building tools, but for someone new, like myself, it would be helpful.